v0x04/utils.py

```diff
@@ -34,7 +33,7 @@ def update_flow_list(controller, switch):
 
 
 def send_desc_request(controller, switch):
-    """Method responsible for request stats of flow to switches.
+    """Responsible for request stats of flow to switches.
```

Comment:
This docstring was copied from `update_flow_list` and I think it's wrong. @beraldoleal, can you suggest a better description for this method?